GroceryStore/InventoryViewController.m

```diff
@@ -0,0 +1,94 @@
+//
+//  InventoryViewController.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "InventoryViewController.h"
+#import "Product.h"
+#import "ProductTableViewCell.h"
+#import "ProductTableView.h"
+
+#import "ProductDetailViewController.h"
+
+@interface InventoryViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (strong, nonatomic) NSArray *productArray;
+@property (weak, nonatomic) IBOutlet ProductTableView *productTableView;
+
+
+@end
+
+@implementation InventoryViewController
+
+- (void)viewWillAppear:(BOOL)animated {
+    
+    [self populateProductArray];
+}
+
+- (ProductTableViewCell *)tableView:(ProductTableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    ProductTableViewCell *productCell = [tableView dequeueReusableCellWithIdentifier:@"productCell" forIndexPath:indexPath];
```

Comment:
ideally, this cell identifier string would be a constant or based on the class (I often use the class name for identifiers and `NSStringFromClass()`)